### PR TITLE
Inputs

### PR DESCRIPTION
**Pull request by coldpigli**
Added Inputs with Error

inputs-png

Added Checkboxes

inputs-checkbox

Added TextArea

text-area
a

### CODE REVIEW

File: css/components/inputs.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    width: var(--fs-lg);
+    height: var(--fs-lg);
+}
+
+.checkbox-wrapper> label{
+    font-size: var(--fs-lg);
+}
+
+.text-box{
+    outline: none;
```
Comment (sadathshariff) [inline css/components/inputs.css]: making outline: none is not a good UX, creates issues for people navigating the web with a keyboard. everything else looks good

**Review by vivekmehto — APPROVED** [review]
Everything seems good to me.

File: css/components/inputs.css (hunk 1 above)
Comment (coldpigli) [reply css/components/inputs.css]: Oh yes. Thanks. Didn't know this.

Comment (coldpigli) [reply css/components/inputs.css]: Removed it.

**Pull request merged**